src/c_string.rs

```diff
@@ -0,0 +1,413 @@
+//! A fixed capacity [`CString`](https://doc.rust-lang.org/std/ffi/struct.CString.html).
+
+use crate::{vec::Vec, CapacityError};
+use core::{
+    ffi::{c_char, CStr},
+    ops::Deref,
+    str,
+};
+
+/// A fixed capacity [`CString`](https://doc.rust-lang.org/std/ffi/struct.CString.html).
+///
+/// It stores up to `N - 1` elements with a byte reserved for the trailing nul terminator.
```

Comment:
Which “elements” are we talking about here? This should be reworded to refer to the length of the string.

Reply:
Updated "elements" as "non-nul characters".